bin/xudcli

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 /* eslint-disable no-console */
 
-const XUClient = require('../lib/xuclient/XUClient');
+const XudClient = require('../lib/xudclient/XudClient');
```

Comment:
I think we should still call this file xucli. "XUD" = "Exchange Union Daemon" and "xucli" = "Exchange Union Command Line Interface." This also matches the naming conventions of bitcoind and btcd for instance.